packages/input-phone/src/PhoneInput.js

```diff
@@ -0,0 +1,309 @@
+            ...tooltipProps,
+          }}
+          disabled={disabled}
+          trigger={
+            <CountrySelectorTrigger
+              disabled={disabled}
+              callingCode={getCountryCallingCode(country)}
+              flag={flags[country]}
+              code={country}
+            />
+          }
+          content={dismiss => {
+            dismissRef.current = dismiss;
+            return (
+              // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
+              <div
+                onClick={event => {
+                  // We stop propagation to avoid giving focus to the main input
+                  // This happens because the popover is inside the input wrapper
+                  event.stopPropagation();
+                }}
+                css="
+                display: flex;
+                flex-direction: column;
+                height: 100%;
+                min-height: 340px;
+                min-width: 300px;
+              "
+              >
+                <FormWrapperHeader
+                  title={textSelectCountryTitle}
+                  cancelLabel={textSelectCountryCancel}
+                  onCancel={dismiss}
+                />
+                <TextInput
+                  placeholder={textSelectCountrySearch}
+                  type="search"
+                  fontSize={fontSizes.SMALL}
+                  margin="8px"
+                  value={countryQuery}
+                  onChange={handleSearchChange}
+                  onKeyDown={handleKeyDown}
+                  innerRef={searchInputRef}
+                  data-testid="country-search-input"
+                />
+                <ul
+                  css={`
+                    padding: 0;
+                    margin: 0;
+                    border-top: solid 1px ${colors.GREY_16};
+                    flex: 1;
+                    width: 100%;
+                  `}
+                >
+                  <AutoSizer>
+                    {({ height, width }) => (
+                      <List
+                        height={height}
+                        itemCount={listData.items.length}
+                        itemData={listData}
+                        itemKey={index => filteredCountries[index].code}
+                        itemSize={54}
+                        width={width}
+                        ref={listRef}
+                      >
+                        {CountryRow}
+                      </List>
+                    )}
+                  </AutoSizer>
+                </ul>
+              </div>
+            );
+          }}
+        />
+      }
+      active={popoverOpen}
+      value={formatIncompletePhoneNumber(phone?.format("NATIONAL"), country)}
+      onChange={handleNumberChange}
+      inputMode="tel"
+      css={`
+        .input {
+          overflow: unset;
+          @media only screen and (max-width: ${fullScreenBreakpoint}) {
```

Comment:
I believe here should be `tooltipProps?.fullScreenBreakpoint || fullScreenBreakpoint`

As `fullScreenBreakpoint` is primarily a `tooltip` prop and we are passing down `tooltipProps` to the tooltip component, it makes sense to check it here